lib/libarbiter/player.cc

```diff
@@ -64,4 +64,22 @@ Player Player::FromJSON(std::string player_string) {
   return Player(player_name, player_elo);
 }
 
+std::string Player::ExportCSV() const {
+  // The name field might include quotes, so we must escape them with another
+  // quote.
```

Comment:
Is there no other "proper" way of escaping string? How would this work for other special characters which are not allowed?